psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -26,31 +26,33 @@
  * @END LICENSE
  */
 
-#include "psi4/fisapt/fisapt.h"
-#include "psi4/fisapt/local2.h"
+#include "fisapt.h"
+
+#include <algorithm>
+#include <functional>
+#include <set>
+#ifdef _OPENMP
+#include <omp.h>
+#endif
 
-#include "psi4/libfock/jk.h"
-#include "psi4/libqt/qt.h"
 #include "psi4/psi4-dec.h"
-#include "psi4/libdiis/diismanager.h"
 #include "psi4/physconst.h"
-#include "psi4/libmints/vector.h"
-#include "psi4/libmints/matrix.h"
+
+#include "psi4/lib3index/dfhelper.h"
+#include "psi4/libcubeprop/csg.h"
+#include "psi4/libdiis/diismanager.h"
+#include "psi4/libfock/jk.h"
 #include "psi4/libmints/basisset.h"
+#include "psi4/libmints/integral.h"
+#include "psi4/libmints/matrix.h"
 #include "psi4/libmints/molecule.h"
 #include "psi4/libmints/potential.h"
-#include "psi4/libmints/integral.h"
+#include "psi4/libmints/vector.h"
 #include "psi4/liboptions/liboptions.h"
-#include "psi4/libcubeprop/csg.h"
 #include "psi4/libpsi4util/process.h"
-#include "psi4/lib3index/dfhelper.h"
-
-#include <ctime>
```

Comment:
I think this header is needed to fix the appveyor compilation.

Reply:
:man_facepalming: